### PR TITLE
[v1.9.x] Make sure files with 2 licenses are listed properly in LICENSE.

### PR DESCRIPTION
**Pull request by josephevans**
Action item 1 from https://github.com/apache/incubator-mxnet/issues/20475.

### CODE REVIEW

File: LICENSE [hunk 1]
```diff
@@ -310,12 +306,16 @@
     python/mxnet/onnx/mx2onnx/_export_onnx.py
     python/mxnet/onnx/mx2onnx/_op_translations/_op_translations_opset12.py
     python/mxnet/onnx/mx2onnx/_op_translations/_op_translations_opset13.py
+    src/operator/numpy/np_einsum_op-inl.h
```
Comment (sandeep-krishnamurthy) [inline LICENSE:309]: should we list functions of BSD clause.

Comment (josephevans) [reply LICENSE:309]: Based on the history of the file, it seems as if it was heavily modified, so I think we are correct to release it under ASF 2.0 license and don't need to call out specific functions.

Comment (josephevans) [reply LICENSE:309]: We also don't call out specific functions in other files in this LICENSE category.

**Review by mseth10 — APPROVED** [review]
LGTM

**Pull request merged**